File: A_JellyfishAndUndertale.cpp
#include <iostream>
using namespace std;

void solve()
{
    int a, b, n;
    cin >> a >> b >> n;
    int arr[n];
    long long int sum = b;
    for (int i = 0; i < n; i++)
    {
        cin >> arr[i];
        arr[i] = min(arr[i], a - 1);
        sum += arr[i];
    }
    cout << sum << endl;
}

int main()
{
    int t;
    cin >> t;
    while (t != 0)
    {
        solve();
        t--;
    }
    return 0;
}

File: A_JellyfishAndUndertale.py
for _ in range(int(input())):
    a,b,n = list(map(int,input().split(" ")))
    if(a > 0 and b > 0 and n > 0 and n<=100):
        arr = sorted(list(map(int,input().split(" "))))
        res = sum(list(map(lambda x: a-1 if x>a-1 else x,arr))) + b 
        print(res)
